charts/fission-core/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+name: fission
+version: 0.1.0
+description: A Helm chart for Fission. Fission is a fast serverless framework for Kubernetes.
+keywords:
+- fission
+- serverless
+home: http://fission.io/
+maintainers:
```

Comment:
Can you add your name and email to this list, and mine as well?

(And I'm not sure Sanket still wants to be contacted about it. @ssudake21, do you still want to be in the list of maintainers for this chart?)